gpflow/likelihoods/multilatent.py

```diff
@@ -112,11 +110,9 @@ def __init__(
         def conditional_distribution(Fs) -> tfp.distributions.Distribution:
             tf.debugging.assert_equal(tf.shape(Fs)[-1], 2)
             loc = Fs[..., :1]
-            scale = transform(Fs[..., 1:] / 2)
+            scale = transform(Fs[..., 1:])
```

Comment:
@vdutor flagging this up as it's changing behaviour - I'm assuming you had the division by 2 so that the latent GP models the log-variance, but that's only true if the transform is actually Exp, so that seemed a bit weird.